roles/jupyterhub/templates/patch_await.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+# add await to asyncio change password function
+
+cat /opt/iiab/jupyterhub/lib/python3.7/site-packages/firstuseauthenticator/firstuseauthenticator.py |grep 'await self.render'
```

Comment:
@georgejhunt doesn't `python3.7` need to be softcoded using {{ python_ver }} so this works on different OS's / versions?

Ideally with `patch_await.sh.j2` if so?

Reply:
If this is true (I don't know, I'm asking?) then Line 7 would need it too of course.